xdsl/interpreters/onnx.py

```diff
@@ -0,0 +1,32 @@
+from xdsl.dialects import onnx
+from xdsl.interpreter import (
+    Interpreter,
+    InterpreterFunctions,
+    PythonValues,
+    ReturnedValues,
+    impl,
+    register_impls,
+)
+
+
+@register_impls
+class OnnxFunctions(InterpreterFunctions):
+    @impl(onnx.Add)
+    def run_add(self, interpreter: Interpreter, op: onnx.Add, args: PythonValues):
+        return ReturnedValues(args), ()
```

Comment:
```suggestion
        lhs, rhs = args
        return lhs + rhs
```

Reply:
actually I guess what we really want to do is
1. check that the inputs are all shaped ptrs
2. create numpy arrays with their contents
3. call + on the numpy arrays
4. return result